catalogue/webapp/pages/item.js

```diff
@@ -123,77 +289,62 @@ const ItemPage = ({
       imageAltText={''}
       hideNewsletterPromo={true}
     >
-      <Layout12>
-        <Paginator
-          currentPage={canvasIndex + 1}
-          pageSize={1}
-          totalResults={canvases.length}
-          linkKey={'canvas'}
-          link={itemUrl({
-            workId,
-            query,
-            page: pageIndex + 1,
-            canvas: canvasIndex + 1,
-            workType,
-            itemsLocationsLocationType,
-            sierraId,
-          })}
-          render={Pagination}
-        />
-      </Layout12>
-      <Layout12>
-        <img
-          className={classNames({
-            'block h-center': true,
-            [spacing({ s: 2 }, { margin: ['bottom'] })]: true,
-          })}
-          style={{
-            maxHeight: '80vh',
-            width: 'auto',
-          }}
-          width={largestSize.width}
-          height={largestSize.height}
-          src={urlTemplate({
-            size: `${largestSize.width},${largestSize.height}`,
-          })}
-        />
-        <Paginator
-          currentPage={pageIndex + 1}
-          pageSize={5}
-          totalResults={canvases.length}
-          linkKey={'page'}
-          link={itemUrl({
-            workId,
-            query,
-            page: pageIndex + 1,
-            canvas: canvasIndex + 1,
-            workType,
-            itemsLocationsLocationType,
-            sierraId,
-          })}
-          render={Pagination}
-        />
-        <div className={classNames({ flex: true })}>
+      <IIIFViewer>
+        <IIIFViewerMain>
+          <Paginator {...mainPaginatorProps} render={XOfY} />
+          <img
+            className={classNames({
+              'block h-center': true,
+              [spacing({ s: 2 }, { margin: ['bottom'] })]: true,
+            })}
+            width={largestSize.width}
```

Comment:
Do we want the largest size here?
Even if we are on mobile?

Reply:
Almost certainly not. But this is very much for user testing, not for going live with. I think we're good for now.

Could maybe add a `FIXME` for everything that will need updating, but in my head the whole file was MVP so maybe it needs one big `FIXME` at the top.